websites/D/Dropout/presence.ts

```diff
@@ -0,0 +1,203 @@
+import { QueryParams, IFrameData } from "./interfaces";
+
+const presence = new Presence({
+		clientId: "1240164154682249227",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	logo = "https://i.ibb.co/hCK6Lj3/dropout-logo.png";
+
+let duration: number, currentTime: number, paused: boolean, playback: boolean;
+
+presence.on("iFrameData", (data: IFrameData) => {
+	playback = data.duration ? true : false;
+	if (playback) ({ duration, currentTime, paused } = data);
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: logo,
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href } = document.location,
+		showButtons = await presence.getSetting<boolean>("buttons"),
+		path = pathname.split("/");
+
+	path.shift();
```

Comment:
?????

Reply:
![image](https://github.com/PreMiD/Presences/assets/25342763/721e3a24-59a2-44af-a0b4-017b81d59bc6)
when i split by `/`, there is an element in there that's empty, i remove it by using `Array.shift()`
![image](https://github.com/PreMiD/Presences/assets/25342763/f40e5b81-e552-4e56-97f7-2f47d8f0780c)

i didn't think this would be a problem, since I did this in my other two presences and it wasn't flagged a problem. Do you have an alternative?

Reply:
Another solution is `.filter(Boolean)`

Reply:
Fixed.